src/ui/debug_tools.rs

```diff
@@ -213,7 +213,7 @@ fn draw_ysort_lines(
         for (ysort, fighter_meta, transform) in query.iter() {
             //If the fighter meta is not loaded default to 16.0
             let half_width = if let Some(meta) = fighter_assets.get(fighter_meta) {
-                meta.size.x / 2.0
+                meta.size.x
```

Comment:
Maybe a simple mention in the `FighterMeta` that the size is actually half of the size.